submissions/sprint_b/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+pytest = "*"
+
+[packages]
+flask = "*"
+python-dotenv = "*"
+pytest = "*"
```

Comment:
pytest doesn't need to be listed twice. Remove it from the all packages section and leave it as a development dependency in the dev-packages section.